src/app/Main/Body/QueryPage/QueryPageContent-Layout.component.tsx

```diff
@@ -15,22 +15,15 @@ import { QueryPageContentBarChartResultsComponent } from "./Results/Charts/Query
 const contentStyle = css`
     width: 0;
     height: inherit;
-    margin-right: 1em;
-    margin-left: 1em;
     padding: 2em;
     flex: 1 1 0;

```

Comment:
I suggest to write a ticket to place all flex properties of flex items to the components where the flex box is defined. E.g. this flex should be in a div surrounding this component. The div should be placed in the QueryPage-Layout component. The reason is that the flex property here controls the behavior of the QueryPage-Layout. The QueryPageContent-Layout component should not depend on where it is used, so not depent on css properties in its parent component.

Reply:
Ticket is written.